v19.1/build-a-java-app-with-cockroachdb-hibernate.md

```diff
@@ -266,7 +266,7 @@ For more information about importing data from MySQL, see [Migrate from MySQL](m
 
 ### Use `rewriteBatchedInserts` for increased speed
 
-We strongly recommend setting `rewriteBatchedInserts=true`; we have seen 2-3x performance improvements with it enabled. From [the JDBC connection parameters documentation](https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters):
+We strongly recommend setting `rewriteBatchedInserts=true`; we have seen 2-3x performance improvements with it enabled. From [the JDBC connection parameters documentation](https://jdbc.postgresql.org/documentation/use/#connection-parameters):
```

Comment:
Oh good. I was looking at the old one and I don't think it was even using the fragment correctly before the link broke.